### PR TITLE
Update WHITEPAPER.md

### PR DESCRIPTION
**Pull request by crypto-rsi**
fix grammar

### CODE REVIEW

**Review by fkimono185 — APPROVED** [review]

**Comment by FAvE01** [comment]
[README.md](https://github.com/binance-chain/whitepaper/files/7558449/README.md)

**Comment by MUM112233** [comment]
gh pr checkout 39

**Comment by MUM112233** [comment]
crypto-rsi:patch-1

**Review by SATOSHI66 — APPROVED** [review]

**Review by EmtMeshal — APPROVED** [review]

**Review by EmtMeshal — APPROVED** [review]

File: WHITEPAPER.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ _NOTE: This document is under development. Please check regularly for updates!_
 - [Outlook](#outlook)
```
Comment (uhug2) [inline WHITEPAPER.md:31]: Jm89* smcl!@gn

**Review by uhug2 — APPROVED** [review]

File: WHITEPAPER.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ _NOTE: This document is under development. Please check regularly for updates!_
...
 # Motivation
 
-After its mainnet community [launch](https://www.binance.com/en/blog/327334696200323072/Binance-DEX-Launches-on-Binance-Chain-Invites-Further-Community-Development) in April 2019, [Binance Chain](https://www.binance.org) has exhibited its high speed and large throughput design. Binance Chain’s primary focus, its native [decentralized application](https://en.wikipedia.org/wiki/Decentralized_application) (“dApp”) [Binance DEX](https://www.binance.org/trade), has demonstrated its low-latency matching with large capacity headroom by handling millions of trading volume in a short time.
+After its mainnet community [launch](https://www.binance.com/en/blog/327334696200323072/Binance-DEX-Launches-on-Binance-Chain-Invites-Further-Community-Development) in April 2019, [Binance Chain](https://www.binance.org) has exhibited its high speed and large throughput design. Binance Chain’s primary focus, its native [decentralized application](https://en.wikipedia.org/wiki/Decentralized_application) (“dApp”) [Binance DEX](https://www.binance.org/trade), has demonstrated its low-latency matching with large capacity headroom by handling millions of trading volumes in a short time.
```
Comment (Fredrexee) [inline WHITEPAPER.md:34]: 368r59BRa4DgFTR9FWTRke52kYYxa7okHz

**Review by Fredrexee — APPROVED** [review]

**Review by Angel0604 — APPROVED** [review]

**Comment by Delegat43** [comment]
bf69b38c39c8f5aa1353c0ac10ced720f1c3ae67

File: WHITEPAPER.md (hunk 2 above)
Comment (atifnimran) [reply WHITEPAPER.md:34]: Ok

Comment (Bobski95) [reply WHITEPAPER.md:34]: I appreciate this so much. Thanks!

**Review by Amarakhaid — COMMENTED** [review]
Aprove in trust wallet BNB smart chain

**Review by brilliant-lx — APPROVED** [review]

**Review by Amarakhaid — APPROVED** [review]

**Pull request closed without merging**